cirq/google/devices/known_devices_test.py

```diff
@@ -498,6 +498,58 @@ def test_sycamore_grid_layout():
         cirq.google.Sycamore23.validate_operation(sqrt_iswap)
 
 
+def test_proto_with_circuitop():
+    circuitop_gateset = cirq.google.serializable_gate_set.SerializableGateSet(
+        gate_set_name='circuitop_gateset',
+        serializers=[cgc.CIRCUIT_OP_SERIALIZER],
+        deserializers=[cgc.CIRCUIT_OP_DESERIALIZER],
+    )
+    circuitop_proto = cirq.google.devices.known_devices.create_device_proto_from_diagram(
+        "aa\naa",
+        [circuitop_gateset],
+    )
+
+    assert (
+        str(circuitop_proto)
+        == """\
+valid_gate_sets {
```

Comment:
Apart from the transitional phase where we're still bringing support online, I think assuming all devices support circuit operations is valid, since in the worst case you can simply decompose the circuit operation.

Reply:
TODO(95-martin-orion): test creating a device spec with CircuitOperation to identify necessary changes / ensure that this is well-behaved.

Reply:
@dstrain115: added `test_sycamore_circuitop_device` to provide this coverage, and it failed. Updated `serializable_device` to resolve the issue.